sklearn/mixture/bayesian_mixture.py

```diff
@@ -781,3 +822,73 @@ def _set_parameters(self, params):
                                       self.precisions_cholesky_.T)
         else:
             self.precisions_ = self.precisions_cholesky_ ** 2
+
+    def partial_fit(self, X, y=None):
+        """Estimate model parameters with the EM algorithm using the posteriors
+        of previous fits as priors.
+
+        The method fit the model `n_init` times and set the parameters with
+        which the model has the largest likelihood or lower bound. Within each
+        trial, the method iterates between E-step and M-step for `max_iter`
+        times until the change of likelihood or lower bound is less than
+        `tol`, otherwise, a `ConvergenceWarning` is raised.
+
+        Parameters
+        ----------
+        X : array-like, shape (n_samples, n_features)
+            List of n_features-dimensional data points. Each row
+            corresponds to a single data point.
+
+        lr : float, default = 1
+
+        Returns
+        -------
+        self
+        """
+        X = _check_X(X, self.n_components)
+        if not hasattr(self, 'converged_'):
+            self._check_parameters(X)
+
+        do_init = not(self.warm_start and hasattr(self, 'converged_'))
```

Comment:
But I'm not sure if this use of warm_start makes sense. This means that the full init is done if `warm_start=False`. I think this is unusual semantics for `partial_fit`.